src/main/java/io/github/pmckeown/dependencytrack/upload/UploadBomRequest.java

```diff
@@ -65,6 +71,18 @@ public List<ProjectTag> getProjectTags() {
         return projectTags;
     }
 
+    public String getParentUUID() {
+        return parentUUID;
+    }
```

Comment:
@pmckeown I would like to but this would break the API contract definition which requires the field named  `parentUUID`:
```
BomSubmitRequest:
      required:
      - bom
      - project
      - projectName
      - projectVersion
      type: object
      properties:
        project:
          pattern: "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$"
          type: string
          example: 38640b33-4ba9-4733-bdab-cbfc40c6f8aa
        projectName:
          pattern: "^[\\p{IsWhite_Space}\\p{L}\\p{M}\\p{S}\\p{N}\\p{P}]*$"
          type: string
          example: Example Application
        projectVersion:
          pattern: "^[\\p{IsWhite_Space}\\p{L}\\p{M}\\p{S}\\p{N}\\p{P}]*$"
          type: string
          example: 1.0.0
        projectTags:
          type: array
          description: Overwrite project tags
          items:
            $ref: "#/components/schemas/Tag"
        autoCreate:
          type: boolean
        parentUUID:
          pattern: "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$"
          type: string
          example: 5341f53c-611b-4388-9d9c-731026dc5eec
        parentName:
          pattern: "^[\\p{IsWhite_Space}\\p{L}\\p{M}\\p{S}\\p{N}\\p{P}]*$"
          type: string
          example: Example Application Parent
        parentVersion:
          pattern: "^[\\p{IsWhite_Space}\\p{L}\\p{M}\\p{S}\\p{N}\\p{P}]*$"
          type: string
          example: 1.0.0
        isLatestProjectVersion:
          type: boolean
        bom:
          pattern: "^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$"
          type: string
          description: Base64 encoded BOM
          example: ewogICJib21Gb3JtYXQiOiAiQ3ljbG9uZURYIiwKICAic3BlY1ZlcnNpb24iOiAiMS40IiwKICAiY29tcG9uZW50cyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAibGlicmFyeSIsCiAgICAgICJuYW1lIjogImFjbWUtbGliIiwKICAgICAgInZlcnNpb24iOiAiMS4wLjAiCiAgICB9CiAgXQp9
```
Would you like me to rename the internal variables and method to `parentUUID`, `getParentUUID()` and `setParentUUID()` instead?

Reply:
Ah fair point. No don’t worry about it.